src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Do not apply any rules to the `*` selector except `box-sizing`

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/207930662-a1085e2c-38f3-486e-9f2d-2c6fe73aa6f9.png)